integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/jms/transport/test/JMSSenderStaleConnectionsTestCase.java

```diff
@@ -86,8 +76,9 @@ public void staleConnectionsTestJMSProxy() throws Exception {
         }
 
         /* restart the JMS broker */
-        stopBroker();
-        startBroker();
+        ActiveMQServer activeMQServer = new JMSBrokerManager().getActiveMQServer();
```

Comment:
Since we are instantiating a new JMSBrokerManager, the returned activeMQserver will be a new one rather than the one which was started. Would it not have an impact?

Reply:
Actually the getActiveMQServer() method is returning the activeMQ server that is already up, not a new one.

Reply:
Yes it would. Fixed the issue.